lib/core/include/irods/objInfo.h

```diff
@@ -151,6 +151,9 @@ typedef struct DataObjInfo {
     int  dataAccessInx;
     int  writeFlag;
     char destRescName[NAME_LEN];
+    // The backupResc feature has been deprecated and will be removed in a future release. It is not being marked as
+    // deprecated for compilation purposes because we cannot change this struct and it is used everywhere.
```

Comment:
We can probably update this comment to indicate that the feature has been removed, and yet we still cannot change the struct.

Reply:
Updated the comment to state the removal of the feature, and the issues with federation if removed.